src/components/molecules/m-mapbox.vue

```diff
@@ -29,12 +30,16 @@ export default {
     for (let i = 0; i < this.data.length; i++) {
       // geojson.features.forEach(function (marker) {
       //   // create a HTML element for each feature
-      //   var el = document.createElement("div");
-      //   el.className = "marker";
+      var el = document.createElement("div");
```

Comment:
please use a better and descriptive name than `el` may be `markerElement`

Reply:
done